base/applications/cmdutils/clip/lang/it-IT.rc

```diff
@@ -0,0 +1,16 @@
+/*
+ * PROJECT:     ReactOS Clip Command
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\nInserisci ""CLIP /?"" per leggere le informazioni sull'uso.\n"
+    IDS_HELP "\nRiindirizza in clipboard l'uscita dei programmi a riga di comando.\n\n\
```

Comment:
"Reindirizza verso la clipboard l'output (o l'esito) [...]" i suppose is better

Reply:
Yes, this one would suffice better than my phrase. Thanks for your suggestion.